cleanlab/count.py

```diff
@@ -425,7 +424,7 @@ class 0, 1, ..., K-1. `pred_probs` should have been computed using 3 (or
     # Estimate the probability thresholds for confident counting
     if thresholds is None:
         # P(we predict the given noisy label is k | given noisy label is k)
-        thresholds = [np.mean(pred_probs[:, k][labels == k]) for k in range(num_classes)]
+        thresholds = get_confident_thresholds(labels, pred_probs, multi_label=multi_label)
     thresholds = np.asarray(thresholds)
```

Comment:
`thresholds = np.asarray(thresholds)` seems unnecessary if the `get_confident_thresholds()` properly returns a np.array like it says it does in its docstring.

Reply:
Leave as is. User could have passed in thresholds... as a list. Note this occurs after the if statement.